src/WrappedVault.sol

```diff
@@ -292,6 +293,7 @@ contract WrappedVault is Owned, ERC20, IWrappedVault {
         uint256 rate = rewardsAfterFee / (end - start);
```

Comment:
Assuming I'm correct about the event argument below, here you can also eliminate `rewardsAfterFee as an intermediate variable:

```diff
-        uint256 rewardsAfterFee = totalRewards - frontendFeeTaken - protocolFeeTaken;
-        uint256 rate = rewardsAfterFee / (end - start);
+        uint256 rate = (totalRewards - frontendFeeTaken - protocolFeeTaken) / (end - start);
```